frontend/src/views/HomePage/creatorNetwork/CreatorNetwork.tsx

```diff
@@ -1,67 +1,182 @@
-import React from "react";
-import homeImage from "assets/homeImage.png";
+import React, { useContext, useEffect, useState } from "react";
 import arrowLeft from "assets/arrowLeft.svg";
 import arrowRight from "assets/arrowRight.svg";
 import style from "./creatorNetwork.module.scss";
-import profilePicture from "assets/profilePicture.png";
 import exampleImage from "assets/exampleImage.png";
-import { ProfileHorizontal } from "components";
+import { Modal, ProfileHorizontal } from "components";
+import { useAppSelector } from "store/store";
+import { AuctionItem } from "interfaces";
+import { Link } from "react-router-dom";
+import { intervalToDuration } from "date-fns";
+import { AddBidModal } from "components/Modal";
+import { addBid } from "API/UserService";
+import { toast } from "react-toastify";
+import { BidOffer } from "interfaces";
+import { SocketContext } from "App";
+import { getAuction } from "API/UserService";
 
 export const CreatorNetwork = () => {
+    const socket = useContext(SocketContext);
+    const [auction, setAuction] = useState<AuctionItem>();
+    const [timeUntillAuctionEnds, setTimeUntillAuctionEnds] = useState<Duration>();
+    const [bidModalVisibility, setBidModalVisibility] = useState(false);
+    const lastBid = auction?.bidHistory[auction.bidHistory.length - 1]?.bid;
+    const user = useAppSelector((state) => state.user);
```

Comment:
Refer to comment in marpuz PR for this one

Reply:
import { UserSelector } from "store/user";
const user = useAppSelector(UserSelector);